src/components/CategoriesContainer.js

```diff
@@ -38,52 +41,33 @@ export const CategoriesContainer = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  const selectCategory = (name) => {
-    dispatch(setIsSelected(name));
-    // setApprovedCount(approvedCount + 1);
-  };
-
   return loading ? (
     <h2>Loading...</h2>
   ) : error ? (
     <h2>{error}</h2>
   ) : (
     <div className='wrapper'>
-      <div className='categoriesContainer'>
-        {categories.map((category, index) => (
-          <div
-            className='category'
-            onClick={() => selectCategory(category.name)}
-          >
-            <button className='status-btn'>
-              {category.isSelected ? (
-                <i className='fa fa-smile-o'></i>
-              ) : (
-                <i className='fa fa-simplybuilt'></i>
-              )}
-            </button>
-
-            <p key={index}>{category.name}</p>
-            <i className='fa fa-info tooltip'>
-              <span className='tooltiptext'>{category.description}</span>
-            </i>
-          </div>
-        ))}
+      <div className='verticalScroll'>
+        <div className='categoriesContainer'>
+          {categories.map((category, index) => (
+            <CategoryItem
+              key={category.name}
+              category={category}
+              index={index}
```

Comment:
You already used the key property for the `map`, you don't need to pass the index to the child components.